src/test/resources/pages/instructorFeedbackResultsEditCommentByDifferentInstructor.html

```diff
@@ -995,7 +995,7 @@ <h4 class="modal-title">
                                     </table>
                                   </div>
                                   <div class="form-group">
-                                    <div class="panel panel-default panel-body content-editor" id="responsecommenttext-1-0-1-1-1" spellcheck="false" style="position: relative;">
+                                    <div class="panel panel-default panel-body content-editor empty" id="responsecommenttext-1-0-1-1-1" spellcheck="false" style="position: relative;">
```

Comment:
This is an unrelated change. It is actually not affected by your code. See #8909 for reference.